.changeset/bright-islands-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+feat: add telemetry to experimental auto-provisioning
```

Comment:
Just a sanity check: This is a patch because it is an experimental feature, right?